doc/DEPENDENCIES.md

```diff
@@ -75,7 +75,7 @@ However, a [CMake build system](BUILDS_AND_CMAKE.md#generating-a-build-system) m
 
 ### Version Requirements
 
-Currently, a version of CMake >= `3.12.0` is required.
+Currently, a version of CMake >= `3.14.0` is required.
```

Comment:
Yeah, anywhere we need to support < 3.14? Looks like our on-prem environment (RHEL7) gets 3.17. @hellkite500 ?

Reply:
Any such environment would have already reported failed configuration if it had Python support activated, since the `NumPy` component didn't exist in earlier versions.